### PR TITLE
Examples/doctests for getting a datetime from an epoch and viceversa.

### PR DESCRIPTION
**Pull request by alatiera**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -299,6 +299,20 @@
 //! [`DateTime.timestamp_subsec_nanos`](./struct.DateTime.html#method.timestamp_subsec_nanos)
 //! to get the number of additional number of nanoseconds.
 //!
+//! ```rust
+//! use chrono::DateTime;
+//! use chrono::Utc;
```
Comment (quodlibetor) [inline src/lib.rs]: Could you hide these two lines behind a `#` doc-hider? So like:

```
//! # use chrono::{DateTime, Utc}
```

**Comment by quodlibetor** [comment]
Thanks for this! I'll push a change up that fixes the new clippy lint which you should rebase onto to get travis to pass.

**Comment by alatiera** [comment]
I will rebase tommorow when I wake up the clippy changes.

File: src/lib.rs [hunk 2]
```diff
@@ -299,6 +299,20 @@
...
+//! use chrono::TimeZone;
+//!
+//! // Construct a datetime from epoch:
+//! let dt = Utc.timestamp(1500000000, 0);
+//! assert_eq!(dt.to_rfc2822(), "Fri, 14 Jul 2017 02:40:00 +0000");
+//!
+//! // Get epoch value from a datetime:
+//! let dt = DateTime::parse_from_rfc2822("Fri, 14 Jul 2017 02:40:00 +0000").unwrap();
+//! assert_eq!(dt.timestamp(), 1500000000);
+//! ```
```
Comment (quodlibetor) [inline src/lib.rs]: Could you add a newline between the ``` and the next section header?

File: src/lib.rs [hunk 3]
```diff
@@ -299,6 +299,20 @@
...
+//! // We need the trait in scope to use Utc::timestamp().
+//! use chrono::TimeZone;
+//!
+//! // Construct a datetime from epoch:
+//! let dt = Utc.timestamp(1500000000, 0);
+//! assert_eq!(dt.to_rfc2822(), "Fri, 14 Jul 2017 02:40:00 +0000");
+//!
+//! // Get epoch value from a datetime:
+//! let dt = DateTime::parse_from_rfc2822("Fri, 14 Jul 2017 02:40:00 +0000").unwrap();
+//! assert_eq!(dt.timestamp(), 1500000000);
```
Comment (quodlibetor) [inline src/lib.rs]: You might need to add underscores in this and the above timestamp to pass clippy (like `1_500_000_000`)

**Comment by alatiera** [comment]
I am not sure If I did the rebase correctly. Feel free to tell me If I need to reset and re-try it.

**Comment by quodlibetor** [comment]
Awesome, thanks!

**Pull request merged**